cmd/provisioner-localpv/app/config.go

```diff
@@ -262,6 +280,32 @@ func (c *VolumeConfig) GetPath() (string, error) {
 		ValidateAndBuild()
 }
 
+func (c *VolumeConfig) IsXfsQuotaEnabled() bool {
+	xfsQuotaEnabled := c.getEnabled(KeyXFSQuota)
+	xfsQuotaEnabled = strings.TrimSpace(xfsQuotaEnabled)
+
+	enableXfsQuotaBool, err := strconv.ParseBool(xfsQuotaEnabled)
+	//Default case
+	// this means that we have hit either of the two cases below:
+	//     i. The value was something other than a straightforward
+	//        true or false
+	//    ii. The value was empty
+	if err != nil {
+		softLimit := strings.TrimSpace(c.getData(KeyXFSQuota, KeyXfsQuotaSoftLimit))
+		hardLimit := strings.TrimSpace(c.getData(KeyXFSQuota, KeyXfsQuotaHardLimit))
+		// Checking if XFS enabling is implied
+		// by the usage of 'KeyXfsQuotaSoftLimit' or
+		// 'KeyXfsQuotaHardLimit'
+		if len(softLimit) > 0 || len(hardLimit) > 0 {
```

Comment:
Is this check still required?

Reply:
In case the Data map is nil, we get an empty string. 

Inside this `if err != nil` block, we don't have a straightforward answer if we want to enable XFS Quota.
The default for XFS Quota is ----> disabled state.

But if someone skips the enabled flag, or messes it up, but has put in Limits, then we can for sure assume he has intentions of enabling XFS Quota.

If the length of the limits are not > 0 AND the 'enabled' value is not straightfowardly 'true' (inside the if err != nil block, there is not straightforward answer). XFS Quota should remain disabled.

Only described limits and explicity enabled: true should enable it.

What do you make of this @kmova ?

Reply:
> Is this check still required?

The default behavior of 0%, 0% will be triggered further down the stack. At this point, empty limits are empty limits.

Reply:
That function is there to judge from the StorageClass config, whether to go into XFS Quota or not. That's all.

Reply:
Disabled implied enabling of XFS Quota. Explicit enable is required to use it now.